packages/block-library/src/table-of-contents/ListLevel.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * WordPress dependencies
+ */
+import { RichText } from '@wordpress/editor';
+
+export default function ListLevel( props ) {
+	const { edit, attributes, setAttributes } = props;
+	let childnodes = null;
```

Comment:
This variable should probably be called `childNodes`. (There are also some variables called `childnode` that should probably be `childNode`.)